MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/annotations/CustomInfoWindowActivity.java

```diff
@@ -67,22 +66,23 @@ public View getInfoWindow(@NonNull Marker marker) {
               ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT));
             parent.setOrientation(LinearLayout.VERTICAL);
 
-            // Depending on the marker title, the correct image source is used. If you
+            // Depending on the marker latitude, the correct image source is used. If you
             // have many markers using different images, extending Marker and
             // baseMarkerOptions, adding additional options such as the image, might be
             // a better choice.
             ImageView countryFlagImage = new ImageView(CustomInfoWindowActivity.this);
-            switch (marker.getTitle()) {
-              case "Spain":
+
+            switch (String.valueOf(marker.getPosition().getLatitude())) {
```

Comment:
That doesn't work because `case R.string.custom_window_marker_title_germany` is an `int` and getTitle() delivers a String. Is there something wrong with using the latitude?